app/assets/stylesheets/modules/_faq_editor.styl

```diff
@@ -0,0 +1,186 @@
+// FAQ Editor Styles
+.faq-editor
```

Comment:
This seems like much more new CSS than should be necessary. We have existing styles from other parts of that app that should be used as much as possible.